src/config/coefficients.json

```diff
@@ -0,0 +1,89 @@
+{
+  "versions": [
+    {
+      "version": 1,
+      "date": "2016-01-01",
```

Comment:
mm, I'd leave it like that, it gives more information about where this value came from (I am guessing it came from an Excel dashboard with that date?)